src/app/banners/BannerWrapper.tsx

```diff
@@ -5,23 +5,23 @@ import { PlanState } from 'app/store/slices/plan';
 import { userSelectors } from 'app/store/slices/user';
 import { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-import SummerBanner from './SummerBanner';
+import Banner from './Banner';
 
-const BannerWrapper = () => {
+const BannerWrapper = (): JSX.Element => {
   const [showBanner, setShowBanner] = useState(false);
   const plan = useSelector<RootState, PlanState>((state) => state.plan);
   const isTutorialCompleted = localStorageService.get(STORAGE_KEYS.SIGN_UP_TUTORIAL_COMPLETED);
   const userPlan = plan.subscription?.type;
   const isNewAccount = useAppSelector(userSelectors.hasSignedToday);
-  const shouldShowBanner = userPlan === 'free' && !localStorageService.get('showSummerBanner');
+  const shouldShowBanner = userPlan === 'free' && !localStorageService.get(STORAGE_KEYS.SHOW_LIFETIME_BANNER);
 
   const onCloseBanner = () => {
     setShowBanner(false);
-    localStorage.setItem('showSummerBanner', 'false');
+    localStorage.setItem('showLifetimeOffer', 'false');
   };
 
   function handleBannerDisplay() {
-    if ((isNewAccount && isTutorialCompleted && shouldShowBanner) || (!isNewAccount && shouldShowBanner)) {
+    if ((isNewAccount && isTutorialCompleted && shouldShowBanner) || shouldShowBanner) {
```

Comment:
This would show the banner before complete the tutorial, it has to work in that way?

Reply:
Once the tutorial is finished. But if the user is not new, shouldShowBanner checks if the user has a free account and showLifetimeOffer value is not in local storage. To make sure that the banner is shown to existing users with a free account @CandelR